src/extension.ts

```diff
@@ -94,10 +94,16 @@ export function activate(context: ExtensionContext) {
         const diagnosticsForFiles: Diagnostic[] =
           convertLinesToDiagnostics(coverage.lines.details);
 
+        let fileName = coverage.file;
+        const is_absolue_path = fileName.length > 2 && (fileName[0] === "/" || fileName[1] === ":");
+        if (!is_absolue_path && workspace.rootPath != undefined) {

```

Comment:
It looks like `rootPath` was deprecated in favor of using the `workspaceFolders`.  Which kind of makes sense, because vscode can have multiple projects open at the same time.
I think higher up the call stack, we know the absolute location of the lcov file, we should find the workspace folder relative to that lcov file.  And then use that as the `rootPath`

Reply:
I filed a PR on your branch with a potential solution:  https://github.com/gilescope/vscode-code-coverage/pull/1